recipe/meta.yaml

```diff
@@ -21,7 +21,8 @@ source:
     - configure.patch
 
 build:
-  number: 0
+  number: 1  # [linux]
+  number: 0  # [win or osx]
```

Comment:
Why not just bump to 1? Seems like it should be ok to have new packages for all of them anyways.